proto/c1/config/config.proto

```
@@ -0,0 +1,90 @@
+    repeated Field fields = 1;
+    repeated Constraint constraints = 2;
+
+    string display_name = 3;
+    string help_url = 4;
+    string icon_url = 5;
+    bool is_directory = 7;
+    string catalog_id = 8;
+}
+
+enum ConstraintKind {
+    CONSTRAINT_KIND_DEPENDENT_ON = 0;
```

Buf (1.47.2)

Enum zero value name "CONSTRAINT_KIND_DEPENDENT_ON" should be suffixed with "_UNSPECIFIED".

(ENUM_ZERO_VALUE_SUFFIX)